src/CmsBundle/DependencyInjection/PermissionRegistry.php

```diff
@@ -75,11 +75,30 @@ class PermissionRegistry
         'USER_CREATE',
     ];
 
+    /**
+     * @param $permission
```

Comment:
If you're using docblocks, please add the variable type, since now it's not very helpful.

See https://docs.phpdoc.org/references/phpdoc/basic-syntax.html.

Also, consider using typehints instead of docblocks, since the minimal php version is already 7.0